modules/sb-psc-attachment/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.20.0"
+      version = ">= 5, <6"
```

Comment:
can we make this >= [4.83.0](https://github.com/hashicorp/terraform-provider-google/releases/tag/v4.83.0)?

Reply:
no, i think the resource is not available before 5 😢

Reply:
according to the release notes linked and the provider doc it should be available as of 4.83

Reply:
i had missed that!!!! so good i will amend immediately. it also makes my life so much easier!!!!

Reply:
p.s. i will update the min version for terraform, since "optional" is supported only from terraform 1.3